src/lib/y2storage/volume_specification.rb

```diff
@@ -408,6 +410,22 @@ def load_features(volume_features)
     # the architecture does not support to resume from swap (i.e., for s390).
     def adjust_features
       self.adjust_by_ram = false if swap? && !resume_supported?
+
+      return unless Y2Storage::BootRequirementsStrategies::Analyzer.bls_bootloader_proposed?
+
+      # Removing grub2/grub2-efi specific subvolumes because they are not needed.
+      # Often these subvolumes are defined in the control.xml file. So, currently it makes
+      # no sense to define it in the SubvolSpecification.fallback_list method because this
+      # will be called only if no entry is in the control.xml file.
```

Comment:
I meant adding a comment to indicate how to avoid removing subvolumes here and  to improve the subvolumes proposal. Something like:

```suggestion
      # Currently, the subvolumes needed for booting are directly defined in the control.xml file (or
      # provided  by the fallback list).  But such subvolumes depend on the selected boot strategy.
      # In the future, each strategy could provide its own list of subvolumes (similar to what happens now
      # with the required partitions for booting). With that, control files do not have to provide the
      # subvolumes for booting and there is no need for removing subvolumes here. 
```